l5kit/l5kit/rasterization/satellite_rasterizer.py

```diff
@@ -51,35 +48,30 @@ def rasterize(
     ) -> np.ndarray:
 
         if agent is None:
-            ego_translation = history_frames[0]["ego_translation"]
+            ego_translation_m = history_frames[0]["ego_translation"]
             # Note 2: it looks like we are assuming that yaw in ecef == yaw in sat image
-            ego_yaw = rotation33_as_yaw(history_frames[0]["ego_rotation"])
+            ego_yaw_rad = rotation33_as_yaw(history_frames[0]["ego_rotation"])
+
         else:
-            ego_translation = np.append(agent["centroid"], history_frames[0]["ego_translation"][-1])
+            ego_translation_m = np.append(agent["centroid"], history_frames[0]["ego_translation"][-1])
             # Note 2: it looks like we are assuming that yaw in ecef == yaw in sat image
-            ego_yaw = agent["yaw"]
+            ego_yaw_rad = agent["yaw"]
 
-        world_to_image_space = world_to_image_pixels_matrix(
-            self.raster_size,
-            self.pixel_size,
-            ego_translation_m=ego_translation,
-            ego_yaw_rad=ego_yaw,
-            ego_center_in_image_ratio=self.ego_center,
-        )
+        raster_from_world = self.render_context.raster_from_world(ego_translation_m, ego_yaw_rad)
+        world_from_raster = np.linalg.inv(raster_from_world)
 
-        # get the center of the images in meters using the inverse of the matrix,
```

Comment:
I'm being very picky here so please ignore if ou think its unnecessary but sometimes you say 'meters' and sometimes 'metres' if there's an easy fix for this (like a control +F in pycharm) maybe make it uniform otherwise ignore

Reply:
I've changed it to `meters` :) Although I prefer the British version (obviously)